src/tools/rbd/action/Import.cc

```diff
@@ -847,7 +848,11 @@ static int do_import(librados::Rados &rados, librbd::RBD &rbd,
     fd = STDIN_FILENO;
     size = 1ULL << order;
   } else {
-    if ((fd = open(path, O_RDONLY)) < 0) {
+    int flag = O_RDONLY;
+    if (sync) {
+        flag |= O_SYNC;
```

Comment:
What does this do for reads? AFAIK, this would have no effect.

Reply:
use sync io so that cgroup blkio speed limit works.

Reply:
Yes, I understand that w/ cgroups v1 you have to disable writeback to get the effect you expect (it's fixed w/ cgroups v2), but what does it mean to SYNC a read? There is nothing to writeback in that case.